src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -156,8 +154,6 @@ object Writer {
         |    ]
         |
         |def maven_dependencies(callback = jar_artifact_callback):
-        |    for hash in list_dependencies():
```

Comment:
Why is this removed?

Reply:
That shouldn't have been changed.
My editor started autoformatting edited files, in the process of undoing this I must have deleted these lines.